runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -3682,6 +3682,15 @@ J9::Z::CodeGenerator::suppressInliningOfRecognizedMethod(TR::RecognizedMethod me
       return true;
       }
 
```

Comment:
We should also have a way to disable this exploitation (May be we can use `feGetEnv` to do that.